source/class.cpp

```diff
@@ -882,6 +882,7 @@ struct ConstructorBindingInfo
 //char const * constructor_if_template = "\tcl.def(\"__init__\", [cl_type](pybind11::handle self_{0}) {{ if (self_.get_type() == cl_type) new (self_.cast<{2} *>()) {2}({1}); else new (self_.cast<{3} *>()) {3}({1}); }}, \"doc\");";
 
 char const * constructor_template =                 "\tcl.def( pybind11::init( []({0}){{ return new {2}({1}); }} ), \"doc\");";
+char const * constructor_template_with_pyArg =      "\tcl.def( pybind11::init( []({0}){{ return new {2}({1}); }} ), \"doc\" {3});";
```

Comment:
sorry, a bit pedantic but i have to keep code tidy: could you please name variable in the same style as rest of the code (ie snake-case), like: `constructor_template_with_py_arg`

Reply:
Done